drf_extra_fields/fields.py

```diff
@@ -180,23 +180,31 @@ def to_internal_value(self, data):
         """
         if html.is_html_input(data):
             data = html.parse_html_dict(data)
+
         if not isinstance(data, dict):
             self.fail('not_a_dict', input_type=type(data).__name__)
+
+        extra_content = list(set(data) - set(["lower", "upper", "bounds", "empty"]))
+        if extra_content:
+            self.fail('too_much_content', extra=', '.join(map(str, extra_content)))
+
         validated_dict = {}
         for key in ('lower', 'upper'):
             try:
-                value = data.pop(key)
+                value = data[key]
             except KeyError:
                 continue
```

Comment:
How about, instead of using `continue` here, we raised something like `ValidationError(f"{key.title()} is required in a range field.")`?

Reply:
I guess it is allowed to have null values, that's why we continue here. Please examine `to_representation` method.